### ISSUE TITLE
LTS won't build with Go 1.7.3

### ISSUE THREAD

**Issue by mboersma** [deis/deis#5108]
As seen on TV at https://ci.deis.io/job/test-ec2/4184/consoleFull#682192135502066c60b2236c09cc9bf8

Need to pin down why this happens now but everything was happy with Go 1.5.x.

**Comment by mboersma** [comment]
There was an ancient version of `golint` installed in the local `$GOPATH`. I worked around this by adding `go get -u github.com/golang/lint/golint` to the job definition, just before it runs the test script.

Now it's failing in a unit test for `deisctl`:
```shell
--- FAIL: TestNewClient (0.00s)
	fleet_test.go:187: failed initializing SSH client: timed out while initiating SSH connection
```

**Comment by mboersma** [comment]
Fixed in #5109.
